infrastructure/template.yaml

```diff
@@ -271,8 +272,10 @@ Resources:
       Timeout: 5
       Environment:
         Variables:
-          DUMMY_TXMA_QUEUE_URL: !FindInMap [TxmaDummyInputQueue, !Ref Environment, QueueUrl]
-          ACCOUNT_MANAGEMENT_URL: !FindInMap [AccountManagementUI, !Ref Environment, url]
+          DUMMY_TXMA_QUEUE_URL:
+            !FindInMap [TxmaDummyInputQueue, !Ref Environment, QueueUrl]
+          ACCOUNT_MANAGEMENT_URL:
+            !FindInMap [AccountManagementUI, !Ref Environment, url]
```

Comment:
This is prettier auto-formatting rather than anything I've intentionally done, but I thought it made sense to leave in.